src/Parser/Monk/Brewmaster/Modules/Features/Abilities.js

```diff
@@ -69,13 +69,13 @@ class Abilities extends CoreAbilities {
         // id
         spell: [SPELLS.IRONSKIN_BREW, SPELLS.PURIFYING_BREW],
         buffSpellId: SPELLS.IRONSKIN_BREW_BUFF.id,
-        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
```

Comment:
These seems like a miscategorization, I recommend using isDefensive instead.

Reply:
Why? The ability is purely defensive.